README.md

```diff
@@ -78,32 +78,31 @@ C++ Installation Instructions
 -----------------------------
 
 0.  Get the source and deal with submodules:
-    + git clone git@github.com:PixarAnimationStudios/OpenTimelineIO.git
-    + cd otio
-    + git checkout cxx
-    + git submodule init
-    + git submodule update
+    + `git clone git@github.com:PixarAnimationStudios/OpenTimelineIO.git`
+    + `cd OpenTimelineIO`
+    + `git submodule init`
+    + `git submodule update`
 
 1. If you want to only build for C++ development (i.e. produce the OTIO C++ libraries and header files), then use cmake:
-    + mkdir build
-    + cd build
-    + ccmake ..
+    + `mkdir build`
+    + `cd build`
+    + `ccmake ..`
```

Comment:
I suggest cmake, since ccmake is only available on platforms with curses installed?